tests/test_utils.py

```diff
@@ -377,3 +388,34 @@ def test_load_spacy_model(model_name, is_found):
     else:
         with pytest.raises(ModuleNotFoundError):
             load_spacy_model(model_name)
+
+
+@pytest.mark.parametrize(
+    "get_obj",
+    [
+        lambda: torch.tensor([8.0, 8.0, 5.0]),
+        lambda: {
+            "a": torch.tensor([8.0, 8.0, 5.0]),
+            "b": torch.tensor([3.0, 1.0, 2.0, 4.0]),
+            "c": torch.tensor([[1.0, 1.9], [1.2, 1.3]]),
+        },
+        lambda: Sequential(Linear(3, 5), Linear(5, 2)),
+    ],
+)
+def test_patched_torch_save(tmpdir, get_obj):
+    md5_sums = []
+
+    for _ in range(2):
+        torch.manual_seed(42)
+        obj = get_obj()
+
+        file_out = pathlib.Path(str(tmpdir)) / "output.pt"
```

Comment:
I guess this one can be moved outside of the `for` loop

Reply:
So here we have to distinguish line-by-line.

### First line
```python
        torch.manual_seed(42)
```
This really cannot go outside the `for` loop, or the `nn.Module` layers will be instantiated with different weights at each iteration, and therefore the output files will differ.

### Second line
```python
        obj = get_obj()
```
If we move this guy _outside_ the `for` loop (together with the previous one) then even w/o the patch the tests would pass! Indeed, calling `torch.save()` twice in a row on the same object has always produced the same binary output—the problem before was with objects that had identical fields but were instantiated at different times (in a way, similar to the difference between `==` and `is`).

### Third line
```python
        file_out = pathlib.Path(str(tmpdir)) / "output.pt"
```
Yes, you're right on this one! Thanks a lot, fixed in 42df3c2

Reply:
Sorry, I was only referring to the path! That was my mistake;)

Reply:
No problem! Feel free to "resolve" if you're satisfied then :)